### ISSUE TITLE
Mount third part ASGI (bentoml) as a sub-application inside fast api and serve

### ISSUE THREAD

**Issue by CodeBreaker444** [tiangolo/fastapi#5686]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import numpy as np
 import bentoml
 from bentoml.io import NumpyNdarray

 iris_clf_runner = bentoml.sklearn.get("iris_clf:latest").to_runner()

 svc = bentoml.Service("iris_classifier", runners=[iris_clf_runner])

 @svc.api(input=NumpyNdarray(), output=NumpyNdarray())
 def classify(input_series: np.ndarray) -> np.ndarray:
     result = iris_clf_runner.predict.run(input_series)
     return result
```

### Description

Trying to mount this app inside the FASTapi as a subapplication with `vnet.mount("/api/v1/ops/",sub_app.svc)`

1. I need to know whether mounting the ASGI app is possible.
2. And does Swagger UI work.

### Operating System

macOS

### FastAPI Version

0.78.0

### Python Version

3.8.13

**Comment by tiangolo** [comment]
I haven't used BentoML myself. Did you try that out? Did it work?

**Comment by CodeBreaker444** [comment]
Yes, As bentoml is also an ASGI based service we can directly mount to fastapi like `app.mount("/api/v1/", bentosvc)` but the problem comes with the openapi documentation and swagger UI as the bentoml uses the same stack they are not working for now but the endpoints are working. As openapi documentation is imp for me in both the services I choose the microservice arch where fastapi acts as a gateway dealing with auth stuff but latency(3x lower than native) between the services has been an issue. Anyway thanks!
